pom.xml

```diff
@@ -181,7 +181,7 @@
             </snapshots>
             <id>ossrh</id>
             <name>oss.sonatype.org-snapshots</name>
```

Comment:
I: while testing the SNAPSHOT release of habushu, discovered that we had some bad configurations for s01.oss.sonatype.org, so fixed those as well.